src/main/java/org/openstreetmap/atlas/utilities/collections/Lists.java

```diff
@@ -0,0 +1,26 @@
+package org.openstreetmap.atlas.utilities.collections;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author lcram
+ */
+public final class Lists
+{
+    @SafeVarargs
+    public static <T> List<T> arrayList(final T... elements)
```

Comment:
I believe this exist in java util as ```Arrays.asList(T ...a)```

Reply:
You're right! I was wondering why we didn't already have this. Removed.